modules/nf-core/sratools/fasterqdump/main.nf

```diff
@@ -40,6 +43,28 @@ process SRATOOLS_FASTERQDUMP {
         ${key_file} \\
         ${sra}
 
+    $exclude_third
+
+    pigz \\
+        $args2 \\
+        --no-name \\
+        --processes $task.cpus \\
+        *.fastq
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        sratools: \$(fasterq-dump --version 2>&1 | grep -Eo '[0-9.]+')
+        pigz: \$( pigz --version 2>&1 | sed 's/pigz //g' )
+    END_VERSIONS
+    """
+
+    stub:
+    def args2       = task.ext.args2    ?:  ''
+    def prefix      = task.ext.prefix   ?:  "${meta.id}"
+    def outfile     = meta.single_end   ?   "${prefix}.fastq" : "${prefix}_1.fastq ${prefix}_2.fastq"
+    """
+    touch $outfile
```

Comment:
I find it convenient in a stub to echo the original command, so it shows what would have been executed.

Reply:
Thanks @Midnighter 

That's a good idea. I have updated to add echo.